src/data/ha-state-history-data.html

```diff
@@ -104,7 +103,9 @@
       this._setIsLoading(true);
 
       data.then(function (stateHistory) {
-        this._setData(stateHistory);
+        // Dom restoration is async and if graph is drown before dom is
```

Comment:
This is not a concern of `ha-state-history-data`. This is instead a concern of whatever element is passing this data through, which would be the `more-info-dialog` who has [exactly this logic](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/dialogs/more-info-dialog.html#L148-L153)